### ISSUE TITLE
Creating customer success message page project intake submission

### ISSUE THREAD

**Issue by RishiRajSahu** [appirio-tech/connect-app#3949]
Currently we show this message for successful submission of an intake request 

<img width="1044" alt="Screenshot 2020-04-22 at 11 01 36 AM" src="https://user-images.githubusercontent.com/11248478/79944283-c2646700-8488-11ea-8c64-82f7a2cbec85.png">

We would need to make this success message configurable via template i.e. we would be having custom message for each kind of intake request.

Further requirements yet to be finalised, creating this issue to keep track of it.

fyi @acshields @vikasrohit

**Comment by vikasrohit** [comment]
@maxceem I think it should be customizable via `metadata` that we store at project template level.

**Comment by maxceem** [comment]
Sure @vikasrohit should we only customize this part of the message? Or any other part?

![image](https://user-images.githubusercontent.com/146016/80082524-2c4e4080-855d-11ea-9d90-ce6f885be8c1.png)

**Comment by vikasrohit** [comment]
Lets customize all three text, heading, subheading and description. All 3 would reside in same object against new key (say `successPage`) in our metadata.

**Comment by maxceem** [comment]
@vikasrohit the issue appears be to not that straightforward as it look like, though we can try to accomplish it in this release anyway.

We have to decide on the approach to handle this:

1. This success page can accessed by the direct link https://connect.topcoder-dev.com/new-project/submitted/9166 - so we can access for any old project. If we want to keep supporting custom messages, then when we open this page we have to implement **loading** of the project (put it somewhere maybe in Redux store), so we can get its Project Template to show custom messages.

2. Alternative solution would be somehow make this page not accessible by the direct link, and after pass the Project Template which has been used for creating project to this view. It's also surprisingly not so trivial as per current code as we remove all the project details as soon as we create it. So we might need to create some place in the Redux Store to hold the project object we've just created, to get the project template for it to display custom messages.

**Comment by vikasrohit** [comment]
I think we can go with option 2 and we even don't need to make the page not accessible. We can just show the default text (as it is right now) if don't have project passed as prop. Now coming to the second part where we are removing the project data from redux store, I am not able to recollect why we are doing that. But that gives me another thought that one of the other solution that came up in our internal discussions was to remove this submitted page and rather automatically take user to the project details page with a message there. If we go with that approach, I think we won't have problem of project loading into the state, right?
Let me discuss that and come back to you.

**Comment by maxceem** [comment]
> If we go with that approach, I think we won't have problem of project loading into the state, right?

Yep. On project details page we are loading project data into the state anyway, so we can just use it.

**Comment by vikasrohit** [comment]
Moving to next release as we don't yet have input from product manager.
